docs/features/logging.md

```diff
@@ -32,9 +42,63 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 }
 ```
 
-If you have multiple middleware components configured, it is advised to put the `ExceptionHandlingMiddleware` as the first one.  By doing so, unhandled exceptions that might occur in other middleware components will also be logged by the `ExceptionHandlingMiddleware` component.
+If you have multiple middleware components configured, it is advised to put the `ExceptionHandlingMiddleware` as the first one.
+By doing so, unhandled exceptions that might occur in other middleware components will also be logged by the `ExceptionHandlingMiddleware` component.
+
+## Logging incoming requests
+
+The `RequestTrackingMiddleware` class can be added to the <span>ASP.NET</span> Core pipeline to log all received HTTP requests.
+The requests by this middleware component are logged through the `ILogger` implementations that are configured inside the project.
+
+The HTTP request headers and request body are by default logged.
```

Comment:
We shouldn't log these by default from a security perspective imo or should add a variety of default headers to omit

Reply:
Was indeed also my concern in the issue. Was not sure how it was eventually resolved.

Reply:
Let's not do this then, what do you think about my suggestion?

Reply:
I think moving it to the log request is an option for the headers, but maybe have an default list of headers to look for (like `Accept`, `Content-Type`, ...), but the body is a different issue bc it can be serialized in diff ways. 
Either way, just adding two booleans is not really something I would vote for.

Reply:
Since we're in Arcus and passing functions is not an option, we could create/use an serialization interface for this.

Reply:
I think it'd be good if there's a list of predefined header names to look for which to omit from the logs, or -maybe a better approach- sanitze the values of those 'sensitive' headers.
(I'm thinking on headers like `Authentication`  , apikeys, etc...

Reply:
I like the idea @fgheysels, we'll add a default list; what would be your suggestions?

- `Authentication`
- `X-API-Key`

Anything else? We can still add more in the future ofcourse.

Reply:
Done that.

Reply:
Every header with `key` in the name ?

Reply:
I also added the request header for the client certificate in app services.

Reply:
Monkey, keystroke, donkey, keyboard, whiskey, jockey, smokey, ?😂

Reply:
That would indeed be to restricting.   I think we can start with the 2 headers that Tom mentions and see if we need to extend it later